### PR TITLE
fix issue #2464

### PR DESCRIPTION
**Pull request by wrfly**
combine parsed storage config with config file

### CODE REVIEW

**Comment by wrfly** [comment]
ping @stevvooe 
What's your opinion?

**Comment by wrfly** [comment]
ping @dmcgowan

**Comment by wrfly** [comment]
@dmcgowan 
Hello ?

**Comment by wrfly** [comment]
hello? @dmcgowan

**Comment by dmp42** [comment]
Discussion on #2464

**Comment by wrfly** [comment]
I do not want contribute.

File: configuration/configuration.go [hunk 1]
```diff
@@ -643,5 +661,13 @@ func Parse(rd io.Reader) (*Configuration, error) {
 		return nil, err
 	}
 
+	fileConfig := new(Configuration)
+	// Since `p.Parse` handled the unmarshal error
+	// We don't have to handle it twice
+	yaml.Unmarshal(in, &fileConfig)
+
+	// Combine parsed config with file config
+	config.Storage.combineWith(fileConfig.Storage)
```
Comment (wrfly) [inline configuration/configuration.go]: Just combine the `storage` part for now.

Comment (wrfly) [inline configuration/configuration.go]: Just combine the `storage` part for now.

Comment (wrfly) [reply configuration/configuration.go]: There could be more sections to be "combined"

**Pull request closed without merging**